packages/components/src/HeaderBar/__snapshots__/HeaderBar.snapshot.test.js.snap

```diff
@@ -10966,6 +11294,22 @@ exports[`Storyshots HeaderBar 🎨 [TIC] HeaderBar 1`] = `
                 <button
                   aria-expanded={false}
                   aria-haspopup={true}
+                  aria-label={
```

Comment:
html is not designed to appear in attribute value

Reply:
<img width="1039" alt="capture d ecran 2018-03-28 a 13 19 27" src="https://user-images.githubusercontent.com/18534166/38026079-0986bb3a-328b-11e8-892a-76d4e3fb2a9f.png">

Reply:
👍 Nice catch, I'll fix it